README.md

```diff
@@ -0,0 +1,50 @@
+# eslint-config-orbitdb
+
+The eslint configuration used for orbit-db projects
+
+## Using in your project
+
+Install [eslint](https://eslint.org), the [OrbitDB linting rules](https://github.com/orbitdb/eslint-config-orbitdb) and add them to your project's dev dependencies:
+```
+npm install eslint @orbitdb/eslint-config-orbitdb --save-dev
+```
+
+Create `.eslintrc` file in your project's root directory:
+```
+touch .eslintrc
+```
+
+Edit `.eslintrc` to contain:
+```json
+{
+  "extends": ["@orbitdb/eslint-config-orbitdb"]
+}
+```
+
+Add a `lint` script to your project's `package.json`:
+```json
+"scripts": {
+  "lint": "./node_modules/eslint/bin/eslint.js src/"
+}
+```
+
+*Note: The lint script needs to call "./node_modules/eslint/bin/eslint.js" instead of "eslint" as users may have global installation of eslint which uses *only* the plugins installed globally and will not find @orbitdb/eslint-config-orbitdb module (nor its dependencies). See more info [here](https://github.com/eslint/eslint/issues/6732) or [here](https://github.com/eslint/eslint/issues/1238).*
```

Comment:
Actually, when evaluating scripts, npm adds ./node_modules/.bin/ to the PATH environment variable. So the script can just use eslint directly

Reply:
I believe the problem with that is that the globally installed eslint bin will override the local one (correct me if I'm wrong) and if so, it'll error because the deps that this package needs are not installed *globally*.

Reply:
If you have it installed as a dev dependency `npm` will use the local binary instead of global